general/payment_acquirers/stripe.rst

```diff
@@ -0,0 +1,32 @@
+======
+Stripe
+======
+
+`Stripe <https://stripe.com/>`_ is a United States-based online payment solution
+provider, allowing businesses to accept **credit cards** and other payment methods.
+
+Enable Local Payment Methods
+============================
+
+Local payment methods are payment methods that are only available for certain merchants
+and customers countries and currencies.
+
+The Stripe connector in Odoo supports the integration of the following local payment methods:
+
+- Bancontact
+- EPS
+- Giropay
+- iDeal:
+- Przelewy24 (P24)
+
+To offer a given local payment method to your customers, it must be listed as a supported payment icon in Stripe.
+Go to :menuselection:`Payment Acquirers --> Stripe --> Configuration` and add the desired payment method in
+**Supported Payment Icons**.
```

Comment:
Suggestion:
```suggestion
To enable specific Local Payment Methods with Stripe, list them as supported payment icons. To do so, go
to :menuselection:`Payment Acquirers --> Stripe --> Configuration` and add the payment methods in the
**Supported Payment Icons** field.
```

Reply:
Ok for me but then add "If the desired payment method is already listed, you don't have anything to do.".

The reason is that this page's first purpose is to guide users after the deployment of a bugfix that adds this behavior. Most users won't have anything to do so we shouldn't let them think that they need to change anything when they actually don't.

Reply:
> To enable specific Local Payment Methods with Stripe, list them as supported payment icons.
To do so, go to :menuselection:`Payment Acquirers --> Stripe --> Configuration` and add the desired payment methods in
the **Supported Payment Icons** field. If the desired payment method is already listed, you don't have anything to do.